db/migrate/20161003201337_create_cheeses.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCheeses < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cheeses do |t|
+      t.string :name, null: false
+      t.integer :user_id, null: false
```

Comment:
It might make sense to actually hold off adding this until you have a user table